modules/learn/pages/clusters-and-availability/xdcr-conflict-logging-feature.adoc

```diff
@@ -0,0 +1,541 @@
+  }`
+| All mutations from the chosen scope will be logged in the specific conflict collection defined by bucket, scope, and collection; instead of the original location mentioned in the *ConflictLogging* key.
+|===
+
+[#tunable-replication-settings]
+== Tunable Replication Settings
+
+By default, the xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflict-logger[Conflict Logger] is tuned to handle a low percentage of conflicts.
+
+For retrieving the tunable settings using the REST API, in a conflict logging enabled replication, see xref:rest-api:rest-xdcr-adv-settings.adoc#rest-api-conflict-logging-replication-tunables[Retrieve the tunable settings for Conflict Logging in a replication].
+
+The details of tuning the Conflict Logger is as follows:
+
+|===
+| Setting | Description | Default
+
+| `conflictLogging`
+| The main setting which when non-empty and valid enables logging of conflicts.
+
+For more information, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#conflictlogging-key[conflictLogging Key].
+|
+
+| `cLogSetTimeoutMs`
+| Sets the timeout for SetMeta operation for conflict writing.
+| 5000ms
+
+| `cLogPoolGetTimeoutMs`
+| Timeout for getting connection from connection pool
+| 5000ms
+
+| `cLogNetworkRetryCount`
+| Number of attempts for a single doc when there is a network error
+| 5
+
+| `cLogNetworkRetryIntervalMs`
+| Time interval to sleep between attempts to write the conflict doc on network error
+| 2000ms
+
+| `cLogWorkerCount`
+| Number of workers used by the conflict logger, which actually write conflict documents.
+| 20
+
+| `cLogQueueCapacity`
+| Queue capacity for the conflict logger.
+| 6000
+
+| `cLogMaxErrorCount`
+| Max number of errors encountered by conflict logger in a “Error Window” after which the conflict logger will be hibernated.
+
+For information about hibernation, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#reliability-components-logger[Reliability Components].
+| 10
+
+| `cLogErrorTimeWindowMs`
+| This is the time window in which the number of errors in the conflict logger will be checked.
+| 120000ms
+
+| `cLogReattemptDurationMs`
+| This is the duration for which the conflict logger will be hibernated and after this the logger will be active again.
+
+For information about hibernation, see xref:learn:clusters-and-availability/xdcr-conflict-logging-feature.adoc#reliability-components-logger[Reliability Components].
+| 600000ms
+|===
+
+[#disable-conflict-logging]
+== Disable Conflict Logging
+
+This section explains disabling XDCR Conflict Logging feature.
+
+* *HLV*: The bucket property `enableCrossClusterVersioning` cannot be disabled once it is enabled.
+For more information, see xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning].
+Enabling HLV depends on `enableCrossClusterVersioning`, so HLV also cannot be disabled after it is enabled.
```

Comment:
* *HLV*:  The prerequisite to using the XDCR Conflict Logging feature is to enable the bucket property `enableCrossClusterVersioning` on all of the buckets of the replication topology.  This enables adding or updating the Hybrid Logical Vector (HLV) metadata when the documents are replicated.  Since the bucket property `enableCrossClusterVersioning` cannot be disabled after it has been enabled, the update and maintenance of HLV metadata by XDCR cannot be disabled. For more information, see xref:learn:clusters-and-availability/xdcr-enable-crossclusterversioning.adoc[XDCR enableCrossClusterVersioning].